### ISSUE TITLE
ELRS version 3.4.0 breaks backpack VTX channel selection

### ISSUE THREAD

**Issue by bnmoore** [DeviationTX/deviation#1042]
When I go to change the VTX channel in the menu I can change the band, but not the channel. It is stuck at whatever the ELRS module was set to previously. This is only for ELRS versions 3.4.0 and above, 3.3.2 still works fine.
ELRS module: Betafpv 1W Micro TX
Radio: Jumper T8SG v2 plus
Firmware: deviation-t8sg_v2_plus-v5.0.0-408f8c0

Edit: I did some digging and this change might be the important one. The VTX channel is now uint8.

src/lib/LUA/tx_devLUA.cpp

![image](https://github.com/user-attachments/assets/0bf376a0-7e28-4d18-bbcb-e7a72639eef7)

Edit 2: yeah, this is it. I cloned the ELRS repo v3.4.3, reverted the relevant changes, and the channel selection now works as expected. I did not confirm that it actually sends correctly to the VTX. I'll look into building Deviation.

**Comment by bnmoore** [comment]
I understand the issue now. Number selection GUI does not have a default step value so it doesn't let the selected value change. Text selection uses a step of 1. When I replace param->step with 1 in the code below, it works as expected. Tested on my radio and confirmed it changes the VTX channel.

src/pages/common/_crsfdevice_page.c
```
static const char *value_numsel(guiObject_t *obj, int dir, void *data)
{
    (void)obj;
    crsf_param_t *param = (crsf_param_t *)data;
    u8 changed = 0;

    param->value = (void *)(intptr_t)GUI_TextSelectHelper((intptr_t)param->value,
                                        param->min_value, param->max_value,
                                        dir, param->step, 10*param->step, &changed);
```

**Comment by hexfet** [comment]
Thanks for the fix!  Merged #1043
